RequestManagementSystem/DB/RequestDB.py

```diff
@@ -517,7 +517,7 @@ def getRequestIDsList( self, statusList = None, limit = None, since = None, unti
 
       reqQuery = reqQuery.order_by( Request._LastUpdate )\
                          .limit( limit )
-      requestIDs = [reqIDTuple[0] for reqIDTuple in reqQuery.all()]
+      requestIDs = [ tuple( reqIDTuple ) for reqIDTuple in reqQuery.all() ]
```

Comment:
why ? This is exactly what you don't want. The origina version was giving you 
requestIDs = [1,2,3,4]
Yours gives
requestIDs = [ (1,), (2,), (3,), (4,) ]